demo/add_tenant.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+set -Eeuo pipefail
+
+source .env
+source ./scripts/aether_functions.sh
+
+echo_message "Adding tenant $1..."
+echo_message "Starting kong..."
+start_container kong $KONG_INTERNAL
+echo_message "Starting keycloak..."
+start_container keycloak "${KEYCLOAK_INTERNAL}/auth"
+echo_message "Starting other services..."
+demo/up.sh
+echo_message "Waiting for other services..."
+sleep 20
+
+DC_AUTH="docker-compose -f docker-compose-generation.yml"
+AUTH_RUN="$DC_AUTH run --rm auth"
+
+function create_kc_tenant {
```

Comment:
You copied these methods from different scripts, maybe you can refactor all of this to avoid useless repetition... :grimacing:

Reply:
Yeah, that's probably worth doing now. I can move them all to `aether_functions.sh`